extensions/amp-ad-network-cloudflare-impl/0.1/cloudflare-a4a-config.js

```diff
@@ -23,7 +23,9 @@
  */
 export function cloudflareIsA4AEnabled(win, element) {
   // check for an explicit attribute to enable A4A processing.
-  // This could be just true, if all ads from this network are A4A, or any
-  // other logic based upon configuration, client settings, etc.
-  return !!element.getAttribute('data-a4a') && element.hasAttribute('src');
+  // This defaults to true, but allows disabling A4A for ads known
```

Comment:
This entire function body needs to be replaced with `return true;`. If it ever returns false, the amp-ad extension will try to use the Delayed Fetch implementation of `<amp-ad type=cloudflare>`, and then it will fail because there is no such implementation and no sensible way to write one (Delayed Fetch relies on the existence of a third-party ad tag that can be injected into an iframe). As such, the `data-cf-a4a` attribute has no effect and should be removed.

Also, this function needs to be registered in [_a4a-config.js](https://github.com/ampproject/amphtml/blob/master/ads/_a4a-config.js) or else it won't be recognized by the amp-ad extension.